client/reader/following-edit/index.jsx

```diff
@@ -544,7 +545,12 @@ const FollowingEdit = React.createClass( {
 							? <FollowingEditSortControls onSelectChange={ this.handleSortOrderChange } sortOrder={ this.state.sortOrder } />
 							: null }
 					{ ! hasNoSubscriptions
-							? <Gridicon icon="search" className="following-edit__search" onClick={ this.toggleSearching } />
+							? <Button 
```

Comment:
Nice find @bluefuton 😃 , makes sense to me.

[Using the aria-label attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)
>Use it in cases where a text label is not visible on the screen

Btw, having access to ```this.translate``` by default on each React component is a really nice touch of the repo.